src/DistributedComputations/distributed_architectures.jl

```diff
@@ -58,50 +123,49 @@ Keyword arguments
                   if not for testing or developing. Change at your own risk!
 """
 function Distributed(child_architecture = CPU(); 
-                         topology, 
-                         ranks,
-                         devices = nothing, 
-                         enable_overlapped_computation = true,
```

Comment:
Now the nonhydrostatic and hydrostatic model will always be working in asynchronous mode?

Reply:
Neither is really “supported”, there’s no code that tests the difference between the two anyways. But in a future PR we can add an option in those model constructors if we think users will want to toggle between the two.